backend/websocketConnect.js

```diff
@@ -10,13 +10,14 @@
  */
 'use strict';
 
-function websocketConnect(uri: string, WebSocket?: (val: string) => Object) {
-  WebSocket = WebSocket || window.WebSocket;
```

Comment:
ok - I think it was when I was experimenting with embedding in a webview or somewhere that didn't have a websocket impl.